### DISCUSSION TITLE
Apps folder showing blank icon for uninstalled apps

### DISCUSSION THREAD

**Discussion by bynarie** [Open-Shell/Open-Shell-Menu#1055] [Q&A] [answered]
As you can see in my image, apps folder is showing microsoft edge, with the blank icon, even though I have completely removed edge completely. Is there a folder where these shortcuts are placed or is there a way to correct this? Thanks!

![openshell](https://user-images.githubusercontent.com/9805554/173708108-35e9c950-50f9-423b-9b8e-6334bbd0257b.jpg)

**Comment by blackcrack** [comment]
https://github.com/Open-Shell/Open-Shell-Menu/issues/1056

it's imo not from Open-shell, it's a cache-thing, the other icons be displayed, so can it not be from OpenShell
and you can close the issue.. because the displaying of the icons is a system-thing in my opinion.. but i can learn more ..

thy
best

**Comment by CTVCAM8** [comment, marked as answer]
This has worked for me sometimes, 
but that is when Microsoft Edge is still installed.
It might work for you.

![Clear cached info](https://user-images.githubusercontent.com/44127216/184560468-6e64d9a8-593a-4551-af3c-b369afe325ee.png)



There doesn't seem to be any way to just delete the 
blank white Microsoft Edge shortcut in the Applications folder
(Run,"shell:AppsFolder")
